desktop-src/direct3dhlsl_current/hlsl-language-attributes.md

```diff
@@ -0,0 +1,3 @@
+# Attributes
+
+Attributes describe additional behavior for language features.  These attributes precede language features and are enclosed in square brackets.
```

Comment:
I don't think this is nearly sufficient for attributes.  Ideally, we should break down the different attribute types that HLSL supports, show the syntax (that's the language part), and link to the built-in section for the set of attributes recognized in each location.

For instance:
- Function Attributes: before a function declaration/definition
- Loop Attributes: before a for/do/while statement
- Branch Attributes: before an if statement
- Switch Attributes: before a switch statement
- etc...